### ISSUE TITLE
_id member not detected

### ISSUE THREAD

**Issue by exploide** [landscapeio/pylint-django#43]
Sample code:
```python
class A(Model):
    x = IntegerField()

class B(Model):
    a = ForeignKey(A)
    y = IntegerField()
    def doSomething(self):
        self.y = self.a_id
```
This causes the error "Instance of 'B' has no 'a_id' member". Looks like the implicit _id members are not detected by pylint-django.

**Comment by carlio** [comment]
Fixed in `0.6.1`
